Author: Jan Engelhardt <jengelh@inai.de>
Description: strip quotes if present when reading /etc/os-release.

diff -Naurp plymouth.orig/src/libply-splash-core/ply-text-progress-bar.c plymouth/src/libply-splash-core/ply-text-progress-bar.c
--- plymouth.orig/src/libply-splash-core/ply-text-progress-bar.c	2012-08-31 07:38:32.204162415 +0200
+++ plymouth/src/libply-splash-core/ply-text-progress-bar.c	2012-08-31 07:39:59.292773887 +0200
@@ -135,6 +135,14 @@ get_os_string (void)
           if (pos2 != NULL)
             *pos2 = '\0';
 
+          if ((*pos == '\"' && pos2[-1] == '\"') ||
+              (*pos == '\'' && pos2[-1] == '\''))
+            {
+              pos++;
+              pos2--;
+
+              *pos2 = '\0';
+            }
           asprintf (&os_string, " %s", pos);
         }
       goto out;
